db/migrations/20220308190810_add_user_role_to_project_roles.php

```diff
@@ -0,0 +1,29 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class AddUserRoleToProjectRoles extends AbstractMigration
+{
+    /**
+     * Change Method.
+     *
+     * Write your reversible migrations using this method.
+     *
+     * More information on writing migrations is available here:
+     * https://book.cakephp.org/phinx/0/en/migrations.html#the-change-method
+     *
+     * Remember to call "create()" or "update()" and NOT "save()" when working
+     * with the Table class.
+     */
+    public function change(): void
+    {
+        $this->table('projectsvacantroles')
+            ->addColumn('projectsVacantRoles_visibleToGroups', 'string',[
```

Comment:
Could we make this a `varchar(1000)` in line with CMS pages?

Reply:
Think that's up to the individual database? The base migration uses 'string' and I got a type error for varchar

![image](https://user-images.githubusercontent.com/54247748/157710822-9ff6d1a1-79d5-43f3-a098-a80cb6c1ed52.png)

![image](https://user-images.githubusercontent.com/54247748/157710983-e15ae49a-7f2a-47ae-adab-d2b6d3e7d2b3.png)